### DISCUSSION TITLE
Moving all charts/skins etc from one device to another within a single file

### DISCUSSION THREAD

**Discussion by Invalidcod3** [ppy/osu#24919] [Ideas] [answered]
I'm a college student who plays osu! at home and school, and basically have a PC at home and a laptop in my dormitory.

So every time I have to get back to my home during extremely long holidays, I have to re-download or backup EVERY single charts so that I can play it on my beafy machine smoothly either...

Could osu! have a solution to export or mark down what chart/skin or even rule set has installed on this machine (and you can select what you want to export) , and then write all of them into a single file. Then I only have to transfer that file onto my main machine, run it through osu!, waiting it to load from the file or downloaded it from the internet, instead of backup a copy or drag and drop osu!'s game file and hoping it works fine.

**Comment by peppy** [comment, marked as answer]
You can transfer the osu! folder (open it from settings via the "open osu! folder button". You can zip the folder to back it up if you want it in a single file.

![osu! 2023-09-25 at 05 58 29](https://github.com/ppy/osu/assets/191335/db24a7b8-299e-4628-9907-34a13b6d21de)

**Reply by Invalidcod3** [reply]
OK, but I hope it could be a more mindless feature, rather than manually checking things I have to move etc.

**Reply by peppy** [reply]
You could write a script for it. Or use a backup program (there are many of these which will do this kind of task for you).

**Reply by Invalidcod3** [reply]
Thanks anyway
